src/Database/Validator/Structure.php

```diff
@@ -151,13 +151,13 @@ static public function getFormat(string $name, string $type): array
     {
         if(isset(self::$formats[$name])) {
             if(self::$formats[$name]['type'] !== $type) {
-                throw new Exception('Format ("'.$name.'") not available for this attribute type ("'.$type.'")');
+                throw new DatabaseException('Format ("'.$name.'") not available for this attribute type ("'.$type.'")');
```

Comment:
We dont need to expose these errors since a `format` is an internal database concept and the end-user can't really do anything with that this information. These exceptions are better just logged.